src/css/main.css

```diff
@@ -1 +1,132 @@
-/* General rules */
+*{
+    margin: 0;
+
+}
+header{
+    text-align: center;
+    height: 96vh;
+    border: 1px solid transparent;
+    background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 50% 100% no-repeat;
+}
+header h1{
+    color:#fff;
+    font-size: 3rem;
+    /* font-family: Helvetica; */
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.